src/theme/ThemeProvider.js

```diff
@@ -55,6 +60,9 @@ const nwTheme = {
   opacity: {
     disabled: 0.5,
   },
+  mediaQueries: {
+    maxWidth: maxWidthMediaQueries,
```

Comment:
Could we have these be object members instead of a function? Maybe we can still use the constants in utility (map on them to create the entries for the maxWidth object). Also I don't think we need the constants in utility, they can live in the theme

Reply:
oh ok ya i see